core/src/main/kotlin/no/ntnu/beardblaster/screen/BeardBlasterStage.kt

```diff
@@ -3,15 +3,15 @@ package no.ntnu.beardblaster.screen
 import com.badlogic.gdx.Gdx
 import com.badlogic.gdx.scenes.scene2d.Stage
 import com.badlogic.gdx.utils.viewport.FitViewport
-import no.ntnu.beardblaster.worldHeight
-import no.ntnu.beardblaster.worldWidth
+import no.ntnu.beardblaster.HEIGHT
+import no.ntnu.beardblaster.WIDTH
 
-class BeardBlasterStage : Stage(FitViewport(worldWidth, worldHeight)) {
+class BeardBlasterStage : Stage(FitViewport(WIDTH, HEIGHT)) {
```

Comment:
Not a big thing, but perhaps use "WORLD_HEIGHT" and "WORLD_WIDTH" to make it more explicit?

Reply:
Agree 👍🏻